AsyncSwift/Models/JSONResponse.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  EventResponse.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import Foundation
+
+struct JSONResponse: Codable {
```

Comment:
```suggestion
struct GetEventResponse: Codable {
```

모든 JSONResponse를 담는 것이 아니라, 
Event 를 get 할 때의 Response model의 구조만 갖고있는 것 같아서 분리하면 좋을 것 같습니다.

Reply:
확인했습니다 수정하겠습니다

Reply:
동사 원형으로 시작하는 네이밍은 함수랑 착각하게 만들 수 있을것같아요! 우디 어찌생각하나요?

Reply:
아니면 어차피 지금 Event 안에 sessions가 들었으니까 json을 수정해서 바로 event struct에 맞게 수정하는거 어떨까요?

Reply:
추후에 TicketingView 혹은 StampView 에서 필요한 내용들 지금 이 json에 모두 담으실껀가요?

Reply:
> 추후에 TicketingView 혹은 StampView 에서 필요한 내용들 지금 이 json에 모두 담으실껀가요?

별도로 생각하고 있었어요 ~ !

Reply:
좋습니다